handlers/attachment_resource.go

```diff
@@ -159,3 +159,50 @@ func HandleGetAttachmentDetails(svc dao.Service) http.Handler {
 		utils.WriteJSONWithStatus(w, req, attachmentResponse, http.StatusOK)
 	})
 }
+
+func HandleDeleteAttachment(svc dao.Service) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		vars := mux.Vars(req)
+		transactionID := utils.GetTransactionIDFromVars(vars)
+		attachmentID := utils.GetAttachmentIDFromVars(vars)
+		if transactionID == "" {
+			log.ErrorR(req, fmt.Errorf("there is no transaction ID in the URL path"))
+			m := models.NewMessageResponse("transaction ID is not in the URL path")
+			utils.WriteJSONWithStatus(w, req, m, http.StatusBadRequest)
+			return
+		}
+
+		if attachmentID == "" {
+			log.ErrorR(req, fmt.Errorf("there is no attachment ID in the URL path"))
+			m := models.NewMessageResponse("attachment ID is not in the URL path")
+			utils.WriteJSONWithStatus(w, req, m, http.StatusBadRequest)
+			return
+		}
+
+		log.InfoR(req, fmt.Sprintf("start DELETE request for attachment with transaction id: %s, attachment id: %s", transactionID, attachmentID))
+
+		responseType, err := service.DeleteAttachment(attachmentID, req)
+		if err != nil {
+			log.ErrorR(req, fmt.Errorf("error deleting attachment: [%v]", err), log.Data{"service_response_type": responseType.String()})
+
+			status, err := utils.ResponseTypeToStatus(responseType.String())
+			if err != nil {
+				w.WriteHeader(http.StatusInternalServerError)
+				return
+			}
+			w.WriteHeader(status)
+			return
+		}
+
+		// Delete attachment from Mongo
```

Comment:
The handler doesn't know which DB is used
```suggestion
		// Delete attachment from DB
```